ccd-definition/CaseEventToFields/draftCMO.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "draftCMO",
+    "CaseFieldID": "cmoHearingDateList",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": 1,
```

Comment:
we can have more meaningful pageIDs, sam mentioned to me that they don't have to be 1, 2, 3 etc, so maybe an ID that sums up or describes what the page does - not too important though 👍

Reply:
Yeah I intend to! Need FPLA-3 in before I can polish them off :)

Reply:
I think numbers are fine for now :)